regular/coc.html

```diff
@@ -0,0 +1,160 @@
+---
+title: HackSoc Code of Conduct
+---
+
+<h2>Values Statement</h2>
+<p>
+    HackSoc aims to be an inclusive space for all, and this code of conduct aims to reflect that. We are not able to cover every single behaviour in this code of conduct, so do not take it as a bullet pointed list of what you can and cannot do. Please follow the spirit of this code as well as the letter. 
+</p>
+<p>
+    If you see behaviour that is unacceptable and the person committing it doesn’t stop when asked, please reach out to a committee member, either in a channel or privately. (Your confidentiality will be respected if you reach out privately.)
+</p>
+<p>
+    We will not tolerate any use of this code of conduct as a hammer against marginalised people speaking out against harassment or microaggressions, or enforcing reasonable boundaries. 
+</p>
+<h2>Rules</h2>
+<h3>General</h3>
+<p>
+<ul>
+    <li>No sexism, racism, ableism, queerphobia, xenophobia, casteism, antisemitism or discrimination of any kind</li>
+    <li>No advocating, even in jest, for Nazis/white supremacy etc</li>
+    <li>Be wary of “dog whistles”, or innocuous-seeming words or images that have another meaning to a specific audience (the <a href="https://www.adl.org/hate-symbols" target="_blank">Anti-Defamation League</a> has a list of (mainly white supremacist) hate symbols which may be used as dog whistles)</li>
+    <li>Avoid making fun of other people’s spelling, word choices, phrasing etc. – if you’re making a joke, think for a few seconds about why it’s funny first</li>
+    <li>Committee may ask you to stop any behaviour that is not explicitly disallowed here</li>
+    <li>No harassment: if someone asks you to stop interacting with them, do not use our spaces to circumvent this</li>
+    <li>This Code of Conduct applies to all HackSoc spaces, online or physical, public or private</li>
+    <li>As a YUSU society, our members are also bound by the <a href="https://yusu.org/about-us/documents/by-laws" target="_blank">YUSU Code of Conduct</a></li>
+</ul>
+</p>
+<h3>Chat Platforms</h3>
+<p>
+<ul>
+    <li>
+        No spam
+        <ul>
+            <li>e.g. using excessive numbers of animated emoji</li>
+        </ul>
+    </li>
+    <li>Bots are welcome, so long as they don’t produce spam</li>
+    <li>Follow the content requirements of those who host us</li>
+    <li>You may be asked to stop a conversation by committee</li>
+    <li>Committee reserve the right to remove messages if they are in violation of the codes of conduct and you have been asked to stop by a moderator</li>
```

Comment:
```suggestion
    <li>Committee reserve the right to remove messages if they are in violation of the codes of conduct and you have been asked to stop by a committee member</li>
```

whoops, i think i just forgot to change this one (but shout me down if leaving it as "moderator" was intentional)